yarn.lock

```diff
@@ -126,16 +126,16 @@
     "@babel/types" "^7.0.0"
 
 "@babel/helper-module-transforms@^7.1.0", "@babel/helper-module-transforms@^7.4.4":
-  version "7.4.4"
-  resolved "https://registry.yarnpkg.com/@babel/helper-module-transforms/-/helper-module-transforms-7.4.4.tgz#96115ea42a2f139e619e98ed46df6019b94414b8"
-  integrity sha512-3Z1yp8TVQf+B4ynN7WoHPKS8EkdTbgAEy0nU0rs/1Kw4pDgmvYH3rz3aI11KgxKCba2cn7N+tqzV1mY2HMN96w==
+  version "7.5.5"
```

Comment:
Hmm, it seems surprising to me that the `yarn.lock` file would have changed so much from adding the `stringify` dependency. I'm guessing that at some point you removed and re-generated the `yarn.lock` file while trying to resolve the Jenkins issue? Can you reset the `yarn.lock` file to the version from the commit where you branched from master (`git checkout <commit> yarn.lock`) and then re-run `yarn install` which should then only apply the changes related to the `stringify` dependency.